wwwroot/src/components/WhyUs.js

```diff
@@ -2,15 +2,175 @@ import React from 'react';
+    margin: `0 auto ${theme.spacing(8)}px`,
+    
+    backgroundImage: `url("${bannerImg}")`,
+    backgroundSize: '1440px 961px',
+    backgroundPosition: 'center 42%'
+  },
+
+  mainSection: {
+    width: '100%',
+    margin: `${theme.spacing(8)}px 0 ${theme.spacing(19.5)}px`,
+    padding: `${theme.spacing(2.5)}px ${theme.spacing(11.5)}px`,
+    [theme.breakpoints.down('sm')]: {padding: `${theme.spacing(2)}px ${theme.spacing(8)}px`}
+  },
+  mainCopyBold: {
+    '& span.boldMe': {
+      fontWeight: 800
+    }
+  },
+  mainCopy: {
+    fontSize: '18px',
+    lineHeight: '35px',
+    paddingRight: theme.spacing(8),
+    [theme.breakpoints.down('sm')]: {paddingRight: 0},
+
+    '& span.orangeMe': {
+      fontSize: '32px',
+      lineHeight: '45px',
+      color: theme.palette.primary.main,
+      fontWeight: 600,
+      '& span.boldMe': {fontWeight: 800}
+    },
+  },
+  copyImg: {
+    width: '112%',
+    position: 'relative',
+    left: -theme.spacing(7),
+    height: theme.spacing(50),
+
+    backgroundImage: `url("${gettingImg}")`,
+    backgroundSize: '726px 1089px',
+    backgroundPosition: 'center 31%',
+
+    [theme.breakpoints.down('md')]: {
+      marginTop: theme.spacing(7),
+      width: '100%',
+      position: 'static'
+    },
+  }
+}))
+
+function BoldCopy({string}) {
+  
+  return (
+    <span className='orangeMe'>
+      {
+        // check to see if any words in the string are wrapped in underscores
+        string.match(/_\S+_/gi).length === 0 ? 
+          // if no underscored words, treat everything as normal
+          string
+        :
+          // split string by underscored words, also capturing underscored words, into array
+          string.split(/(_\S+_)/).map((segment, index) => (
+            segment[0] !== '_' ? 
+              // return non-underscored strings as normal
+              segment
+            :
+              // return underscored words in a span container
+              (<span className='boldMe' key={index}>
+                {segment.slice(1, -1)}
+              </span>)
+          ))
+      }
+    </span>
+  )
+}
 
 export default function WhyUs() {
+  const classes = useStyles();
+
   return (
     <Layout>
-      <div>
-        <h3>Why Us</h3>
-      </div>
-      <FeatureCollection/>
+      <Container className={classes.root} maxWidth='xl' disableGutters>
+        <Container className={classes.quoteContainer}>
+          <Typography variant='h2'>
+            {whyUsData.topQuote.body}
+          </Typography>
+
+          <Typography variant='h3'>
+            {whyUsData.topQuote.source}
+          </Typography>
+        </Container>
+
+        <div className={classes.imgBanner}/>
+
+        <Divider/>
+        
+        <Grid container className={classes.mainSection} alignItems='center'>
+          <Grid item md={7} xs={12}>
+
+            <Typography className={classes.mainCopy}>
+              <BoldCopy string={whyUsData.bodyCopy.bold}/>
+              &nbsp;&nbsp;
```

Comment:
I don't fully understand why you are copying data, making it bold and using regex for that? Could you please elaborate?

Reply:
Yes! I know this is a bit roundabout, but it seems like they haven't finalized the text they want to put on this page yet, so I wanted to make it convenient for us to change the copy text in the future. On the last project I worked on, I was asked to make multiple copy edits that were annoying because of the time it took to do mundane formatting tasks.

I also put the source of data for the text at the top of the page so it would be easier to make future text changes